quickstart/tutorial_2-0.md

```diff
@@ -154,7 +154,7 @@ std::cout << "Joint " << jointName << " connects " << parentLinkName
 
 ```
 
-As you might guess, we're getting the `name` attribute of the current `joint_element` and then retrieving its `parent` and `child` elements' values. It is interesting to note all `Get()` method's use cases: `Get()` is a template method that returns the given attribute value or the element value itself (that is, the plain text in between tags) if called with no arguments. To this end, it has been specialized to parse such value accordingly into primitive types (i.e. `double`), common std types (i.e. `std::string`) or [`ignition::math`](http://ignitionrobotics.org/libraries/math) types for the most complex ones (like poses).
+As you might guess, we're getting the `name` attribute of the current `joint_element` and then retrieving its `parent` and `child` elements' values. It is interesting to note all `Get()` method's use cases: `Get()` is a template method that returns the given attribute value or the element value itself (that is, the plain text in between tags) if called with no arguments. To this end, it has been specialized to parse such value accordingly into primitive types (i.e. `double`), common std types (i.e. `std::string`) or [`gazebo::math`](http://gazebosim.org/libraries/math) types for the most complex ones (like poses).
```

Comment:
Since this is for an earlier version of `libsdformat`, having `ignition::math` might make more sense.